README.md

```diff
@@ -1,11 +1,11 @@
-# Watch
+# Watch :eyes:
 
 [![Actions Status](https://github.com/rafaelsq/wtc/workflows/tests/badge.svg)](https://github.com/rafaelsq/wtc/actions)
 [![Go Report Card](https://goreportcard.com/badge/github.com/rafaelsq/wtc)](https://goreportcard.com/report/github.com/rafaelsq/wtc)
 [![GoDoc](https://godoc.org/github.com/rafaelsq/wtc?status.svg)](https://godoc.org/github.com/rafaelsq/wtc)
 [![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://opensource.org/licenses/MIT)
 
-A simple watch files utility  
+**A simple watch files utility **  
```

Comment:
Continua errado. Se você for no víew file, ira perceber que continua bugado.

Reply:
@rafaelsq realizei os ajustes, eram alguns espaços que não estavam pegando o layout do markdown